requirements.txt

```diff
@@ -1,10 +1,12 @@
 azure-storage-file-share==12.13.0
+bson
```

Comment:
I think `bson` is one of the dependencies of `pydantic`. Hence, gets installed automatically. Need to confirm though.

Reply:
@hardboprobot  Could you please check this?

Reply:
Yes, `bson` comes with `motor`. Can you please pin the version here?